jobs/e2e_node/crio/templates/base/20-crio-drop-infra-ctr.conf

```diff
@@ -0,0 +1,18 @@
+[crio.image]
+signature_policy = "/etc/crio/policy.json"
+
+[crio.runtime]
+log_level = "debug"
```

Comment:
I would make this explict and add the true statement here.

Reply:
done